### PR TITLE
ADD: devbuildcosmo to CodeDev.rst

### PR DESCRIPTION
**Pull request by elsagermann**

### CODE REVIEW

File: CodeDev.rst [hunk 1]
```diff
@@ -82,8 +82,24 @@ Set the path to the serialized data (later it will be used in this guide)
 
   SERIALIZE_DATA=</path/to/cosmo>/cosmo/ACC/test/serialize/data/
 
-Compile and Test a Local C++ dycore
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Serialize your own data using devbuildcosmo (in developement might be unstable)
```
Comment (lxavier) [inline CodeDev.rst:85]: replace devbuildcosmo with dev-build
note:remove all might be unstable in the text

File: CodeDev.rst [hunk 2]
```diff
@@ -195,6 +236,19 @@ Finally we can compile a COSMO executable from the working directory
   cd </path/to/cosmo>/
   spack dev-build -i ${COSMO_SPEC} ^/${DYCORE_HASH}
 
+Compile a local version of COSMO using dev-build (might be unstable)
```
Comment (lxavier) [inline CodeDev.rst:239]: remove this part

File: CodeDev.rst [hunk 3]
```diff
@@ -208,16 +262,28 @@ First we need to find the hash of the C++ dycore installation for the desired va
   DYCORE_SPEC="cosmo-dycore@master real_type=float build_type=Release +cuda"
   DYCORE_HASH=$(spack find --format "{hash}" ${DYCORE_SPEC})
 
+!!!
+Careful here: the spack find command can find more than one version satisfying the DYCORE_SPEC in that case the DYCORE_HASH variable will be a list which you don't want, we therefore adivce to always check the type of the variable:
```
Comment (lxavier) [inline CodeDev.rst:266]: advice

echo $DYCORE_HASH 
(should have only one hash)

File: CodeDev.rst [hunk 4]
```diff
@@ -195,6 +236,19 @@ Finally we can compile a COSMO executable from the working directory
...
+
+  COSMO_SPEC="cosmo@dev-build%pgi real_type=float cosmo_target=gpu +cppdycore +claw"
+
+.. code-block:: bash
+
+  cd </path/to/cosmo>
+  spack dev-build --test=root $COSMO_SPEC
+
 
 Compile cosmo against a master/release version of the dycore
```
Comment (lxavier) [inline CodeDev.rst:253]: remove this part

File: CodeDev.rst [hunk 5]
```diff
@@ -226,6 +292,21 @@ In your working directory of cosmo, compile an executable using spack
   spack dev-build -i ${COSMO_SPEC}
 
 
+Compile cosmo against a master/release version of the dycore with devbuildcosmo (might be unstable)
```
Comment (lxavier) [inline CodeDev.rst:295]: change title

**Comment by elsagermann** [comment]
Deprecated, closing, see new PR #5

**Pull request closed without merging**